### PR TITLE
pi hints

### PR DESCRIPTION
**Pull request by Kdisimone**
i'm sure i'll have more to add, but this is a start

### CODE REVIEW

File: docs/docs/Build Your Rig/pi-install.md [hunk 1]
```diff
@@ -61,6 +61,21 @@ You'll be prompted to set a password.  You'll want to change it to something per
...
 
-The script will then continue to run awhile longer (~10+ minutes) before asking you to press `enter` to run oref0-setup.
+The script will then continue to run awhile longer (~10+ minutes) before asking you to press `enter or control-c` for the setup script options.
+**************************
+At this time, the master installation of oref0 is not compatible with the pi0 installations.  So, instead of proceeding with the setup script, press `control-c` to cancel the setup script.
 
-Return to the [OpenAPS Install page](http://openaps.readthedocs.io/en/latest/docs/Build%20Your%20Rig/OpenAPS-install.html#step-4-setup-script) to complete oref0-setup.
+Reboot your rig by entering `reboot`.  This will end your ssh session.  Give your rig time to reboot and then login to the rig again `ssh root@yourrigname.local` (or Putty equivalent for Windows users).
+
+Now we will select a pi0-compatible branch by using `cd src/oref0 && git checkout 0.7.0-refactor`. You should see a message returned of "Branch 0.7.0-refactor set up to track remote branch 0.7.0-refactor from origin.
```
[the remark this answers is not in the document]
Comment (Kdisimone) [reply docs/docs/Build Your Rig/pi-install.md]: ah, good to know!  I'll correct that.   I have more to add still too.  might take a little time

[the remark this answers is not in the document]
Comment (Kdisimone) [reply docs/docs/Build Your Rig/pi-install.md]: shall i change it from pi0 users to HAT users for clarity?

[the remark this answers is not in the document]
Comment (scottleibrand) [reply docs/docs/Build Your Rig/pi-install.md]: It's actually raspbian that introduced the new kernel version that triggered the incompatibility, so probably "Raspberry Pi compatible" is more correct than saying HAT or specifying the Pi Zero.

**Comment by Kdisimone** [comment]
Ok, probably done for now

**Pull request merged**